app/lib/CommandHandler/CommandHandler.cpp

```diff
@@ -154,93 +169,176 @@ void CommandHandler::processString(const char *inString) {
  * When the terminator character (default COMMANDHANDLER_DEFAULT_TERM) is seen, it starts parsing the
  * buffer for a prefix command, and calls handlers setup by addCommand() member
  */
-void CommandHandler::processChar(char inChar) {
-  Serial.print(inChar);   // Echo back to serial stream, more user friendly.
-  if (inChar == term1 || inChar == term2) {     // Check for the terminators (default: '\n' and '\r') meaning end of command
-    #ifdef COMMANDHANDLER_DEBUG
-      Serial.print("Received: ");
-      Serial.println(buffer);
-    #else
-      Serial.printf("Received Command: %s\r\n", buffer);
-    #endif
-
-    char *command = strtok_r(buffer, delim, &last);   // Search for command at start of buffer
-    if (command != NULL) {
+void CommandHandler::processChar(char inChar)
```

Comment:
repeated code in many places, please refactor this method

Reply:
probably solved I'm not so sure but I hope so